app/javascript/packs/Components/Header.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styles from "./Header.module.scss";
+
+
+class Header extends React.Component {
+
+    render() {
+        return <React.Fragment>
+            <div className={styles['header_image_container']}>
+                <div className={styles['pizza']}/>
+            </div>
+            <div className={styles['header']}>
```

Comment:
You can access these with dot-notation: `styles.header`, `styles.pizza`, `styles.header_image_container`.

You _wouldn't_ be able to do that if, for example, for something like this: `header-image-container`, then you'd have to do `styles['...']`.